get_era5.py

```diff
@@ -0,0 +1,157 @@
+import cdsapi
+import yaml
+import pandas as pd
+from pathlib import Path
+
+c = cdsapi.Client()
+
+# Read case configuration
+with open("cases/era5_2021.yaml") as f:
+    config = yaml.safe_load(f)
+    
+# Use the preprocessing dates to select which dates to download
+datelist = pd.date_range(
+    start=config["preprocess_start_date"], end=config["preprocess_end_date"], freq="d"
+)
```

Comment:
perhaps we can re-use some code from the preprocessing module:

```suggestion
from wam2layers.preprocessing.era5 import parse_config
config=parse_config("cases/era5_2021.yaml")
```

Eventually I think it would be nice to have "parse_config" in a separate module, like `wam2layers.utils`.